packages/device_info/device_info/example/lib/main.dart

```diff
@@ -11,6 +11,9 @@ import 'package:flutter/material.dart';
 import 'package:flutter/services.dart';
 import 'package:device_info/device_info.dart';
 
+import 'package:device_info_platform_interface/device_info_platform_interface.dart';
+import 'package:device_info_platform_interface/model/android_device_info.dart';
```

Comment:
Don't import from the model. This should come from the device_info_platform_interface.dart file.

Reply:
Here: https://github.com/flutter/plugins/blob/master/packages/device_info/device_info_platform_interface/lib/device_info_platform_interface.dart#L14